src/TheGuardianIn2020Banner/index.tsx

```diff
@@ -50,6 +50,28 @@ export const TheGuardianIn2020Banner: React.FC<Props> = ({
 }: Props) => {
     const [showBanner, setShowBanner] = useState(true);
 
+    const onClick = (internalButtonId: number): void => {
+        setShowBanner(false);
+
+        catchAndLogErrors('ophanButtonClick', () => {
+            // Braze displays button id from 1, but internal representation is numbered from 0
+            // This ensures that the Button ID in Braze and Ophan will be the same
+            const externalButtonId = internalButtonId + 1;
+            submitComponentEvent({
+                component: {
+                    componentType: 'RETENTION_ENGAGEMENT_BANNER',
+                    id: ophanComponentId,
+                },
+                action: 'CLICK',
+                value: externalButtonId.toString(10),
+            });
+        });
+
+        catchAndLogErrors('brazeButtonClick', () => {
+            logButtonClickWithBraze(internalButtonId);
+        });
```

Comment:
I _think_ this section is identical between the close click handler and this handler. Think we should refactor it out to a function that both can share?

Reply:
Good point - yes that makes sense.